### PR TITLE
[Jira-v3] - fix issue with summary argument in jira-create-issue command

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-31443)

## Description
See issue's description

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV3/JiraV3_test.py [hunk 1]
```diff
@@ -699,9 +699,42 @@ def test_create_issue_command(self, mocker):
...
+        When
+            - Calling the create issue command.
+        Then
+            - Validate that DemistoException is raised
+        """
+        from JiraV3 import create_issue_command
+        client = jira_base_client_mock()
+        with pytest.raises(DemistoException):
+            create_issue_command(client=client, args={})
+
```
Comment (anas-yousef) [inline Packs/Jira/Integrations/JiraV3/JiraV3_test.py:737]:
```suggestion
        with pytest.raises(DemistoException) as e:
            create_issue_command(client=client, args={})
         assert 'The summary argument must be provided' in str(e)
```

File: Packs/Jira/ReleaseNotes/3_1_6.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Atlassian Jira v3
+
+- Fixed an issue where the **jira-create-issue** command required the *summary* argument even when its passed through the *issue_json* argument.
```
Comment (anas-yousef) [inline Packs/Jira/ReleaseNotes/3_1_6.md:6]:
```suggestion
- Fixed an issue where the **jira-create-issue** command required the *summary* argument even when it is passed through the *issue_json* argument.
```

File: Packs/Jira/Integrations/JiraV3/JiraV3.yml [hunk 3]
```diff
@@ -563,7 +563,7 @@ script:
       hidden: true
     - description: The summary of the issue.
       name: summary
-      required: true
+      required: false
```
Comment (sapirshuker) [inline Packs/Jira/Integrations/JiraV3/JiraV3.yml:566]: Please change it in the readme file

File: Packs/Jira/Integrations/JiraV3/JiraV3.py [hunk 4]
```diff
@@ -1939,6 +1939,8 @@ def create_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> Comman
 
     issue_fields = create_issue_fields(client=client, issue_args=args_for_api,
                                        issue_fields_mapper=client.ISSUE_FIELDS_CREATE_MAPPER)
+    if "summary" not in issue_fields.get("fields", {}):
+        raise DemistoException('The summary argument must be provided')
```
Comment (sapirshuker) [inline Packs/Jira/Integrations/JiraV3/JiraV3.py:1943]:
```suggestion
        raise DemistoException('The summary argument must be provided.')
```

File: Packs/Jira/Integrations/JiraV3/JiraV3.py [hunk 5]
```diff
@@ -1939,6 +1939,8 @@ def create_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> Comman
 
     issue_fields = create_issue_fields(client=client, issue_args=args_for_api,
                                        issue_fields_mapper=client.ISSUE_FIELDS_CREATE_MAPPER)
+    if "summary" not in issue_fields.get("fields", {}):
```
Comment (sapirshuker) [inline Packs/Jira/Integrations/JiraV3/JiraV3.py:1942]: I think that you need to check if "summary" not in issue_fields.get("fields", {}) or not demisto.args()["summary"]
Am I correct?

**Review by sapirshuker — COMMENTED** [review]
Great!!!

File: Packs/Jira/Integrations/JiraV3/JiraV3.yml (hunk 3 above)
Comment (GuyAfik) [reply Packs/Jira/Integrations/JiraV3/JiraV3.yml:566]: nice

Comment (GuyAfik) [reply Packs/Jira/Integrations/JiraV3/JiraV3.yml:566]: will do

File: Packs/Jira/Integrations/JiraV3/JiraV3.py (hunk 5 above)
Comment (GuyAfik) [reply Packs/Jira/Integrations/JiraV3/JiraV3.py:1942]: what @anas-yousef said.
It will always be populated in fields dict

**Review by JudahSchwartz — APPROVED** [review]
great!

**Pull request merged**